rcon/player_history.py

```diff
@@ -497,15 +505,15 @@ def remove_flag(
     return player, old_flag
 
 
-def get_player_messages(player_id: str) -> list[PlayerActionType]:
+def get_player_messages(player_id: str) -> list[PlayerActionState]:
     with enter_session() as sess:
         player = sess.query(PlayerID).filter_by(player_id=player_id).one_or_none()
-        actions: list[PlayerActionType] = []
+        actions: list[PlayerActionState] = []
         if player:
             actions = [
                 action.to_dict()
                 for action in player.received_actions
-                if action.action_type == "MESSAGE"
+                if action.action_type == PlayerActionState.MESSAGE.value
```

Comment:
Unless I'm going crazy, `action` is an instance of `rcon.models.PlayersAction`, making `action.action_type` a string. And we're comparing it to a string here.